graveler/graveler.go

```diff
@@ -1158,12 +1169,13 @@ func (g *Graveler) Merge(ctx context.Context, repositoryID RepositoryID, from Re
 		if err != nil {
 			return "", fmt.Errorf("update branch %s: %w", to, err)
 		}
-		return commitID, nil
+		return &CommitIDAndSummary{commitID, summary}, nil
 	})
+	c := res.(*CommitIDAndSummary)
```

Comment:
Same as before, if possible do this after the error check

Reply:
Yup!